docs/contributing-guide.md

```diff
@@ -1,40 +1,33 @@
 ---
-title: "Contributing Guide"
-sidebar_label: "Introduction"
+title: "Introduction"
 help_discord: "https://discord.gg/fBMuQ7FcG6"
 help_github_discussion: "https://github.com/chatwoot/chatwoot/discussions/categories/contributors"
 ---
 
-This guide will help you get started with Chatwoot! Thank you for taking an interest in contributing to Chatwoot. Before submitting your contribution, please make sure to take a moment and read through the following guidelines:
+Thank you for your interest in contributing to Chatwoot. Please take a moment to read through our guidelines before you contribute.
 
-### Getting Started
+### Get started
 
-- Before starting your work, ensure an [issue](https://github.com/chatwoot/chatwoot/issues) exist for it. If not feel free to create one. You can also take a look into the issues tagged [Good first issues](https://github.com/chatwoot/chatwoot/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+first+issue%22).
-- Add a comment on the issue and wait for the issue to be assigned before you start working on it.
-  - This helps to avoid multiple people working on similar issues.
-- If the solution is complex, propose the solution on the issue and wait for one of the core contributors to approve before going into the implementation.
-  - This helps in shorter turn around times in merging PRs
-- For new feature requests, Provide a convincing reason to add this feature. Real-life business use-cases will be super helpful.
-- Feel free to join our [discord community](https://discord.gg/cJXdrwS), if you need further discussions with the core team.
+1. If you've noticed something that needs to be updated, go through our existing [issues](https://github.com/chatwoot/chatwoot/issues) before you create an issue to help avoid duplicate issues.
+Also, look for [good first issues](https://github.com/chatwoot/chatwoot/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+first+issue%22) that can help you get started on easier issues. 
+2. Add a comment to the issue and wait for the issue to be assigned before you can start working on it. This helps avoid multiple people working on similar issues.
+3. For complex solutions, propose the solution on the issue and wait for one of the core contributors to approve it before it can be implemented. This helps in shorter turnaround time in merging pull requests (PRs).
+4. Raise your PRs against the base branch `develop`. We use the [git-flow](https://nvie.com/posts/a-successful-git-branching-model/) branching model.
+Multiple smaller commits are encouraged as you work on a PR as they will be squashed before merging.
 
-### Pull Request Guidelines
```

Comment:
Removed this heading as it can be confused with https://www.chatwoot.com/docs/handbook/engineering/pr-guidelines. Instead I have tried to club together with the section above that tries to list the workflow from creating an issue to ending with a PR.